app/assets/stylesheets/avo.base.css

```diff
@@ -93,6 +93,14 @@ dialog#turbo-confirm {
   @apply bg-transparent;
 }
 
+/* TODO: make content like tailwindcss */
+.floating-row-controls {
+  &:before {
+    content: "";
```

Comment:
I think this can be used like this in Tailwind. test it first pls.
`content-['']`